<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="SnapshotTableView" name="tableView"/>
   </item>
   <item row="1" column="1">
    <widget class="CompareSnapshotTableView" name="compareTableView"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="headerFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,0,0,0,0,0,0,0">
      <item row="0" column="6">
       <widget class="QPushButton" name="compareSnapshotButton">
        <property name="text">
         <string>Compare to Snapshot</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="LiveButton" name="compareLiveButton">
        <property name="text">
         <string>Compare to Live</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <spacer name="headerSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="8">
       <widget class="QPushButton" name="restoreButton">
        <property name="text">
         <string>Restore</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="primarySnapshotLabel">
          <property name="text">
           <string>#####</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="primarySnapshotTitle">
          <property name="text">
           <string>#####</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="secondarySnapshotLabel">
          <property name="text">
           <string>#####</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="secondarySnapshotTitle">
          <property name="text">
           <string>#####</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="7">
       <widget class="QPushButton" name="removeComparisonButton">
        <property name="text">
         <string>Remove Comparison</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SnapshotTableView</class>
   <extends>QTableView</extends>
   <header>superscore.widgets.page.restore</header>
  </customwidget>
  <customwidget>
   <class>LiveButton</class>
   <extends>QPushButton</extends>
   <header location="global">superscore.widgets.page.restore</header>
  </customwidget>
  <customwidget>
   <class>CompareSnapshotTableView</class>
   <extends>QTableView</extends>
   <header>superscore.widgets.page.restore</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
